spec/swagger/definitions/hand_coded_paths.yaml

```diff
@@ -489,9 +489,10 @@
               $ref: put_move_responses.yaml#/422
 /moves/{move_id}/events:
   post:
-    summary: '[NOT YET IMPLEMENTED] Creates a move event'
+    deprecated: true
+    summary: '[DO NOT USE] Creates a move event'
     description: |
-      ## [NOT YET IMPLEMENTED]
+      ## [DO NOT USE]
```

Comment:
If this wasn't implemented previously, and is now deprecated is it realistic that anyone is actually using it? If not  used then could it be deleted now?

Reply:
alas, it is actually being used by the front end, as a temporary endpoint, so can't be deleted right away.